cmake/Modules/winconf-mingw.cmake

```diff
@@ -1,10 +1,3 @@
-if (STACKTRACE)
```

Comment:
Is it because `STACKTRACE` are not available here?
If so I think a better fix is to move 
```
if(WIN32)
    include(winconf)
endif(WIN32)
```
in https://github.com/qbittorrent/qBittorrent/blob/master/CMakeLists.txt#L42-L44
to just above 
```
add_subdirectory(src)
add_subdirectory(dist)
```
Can you try it? Also ping @jagannatharjun, could you please try it too?

Reply:
I wouldn't place these project-related settings outside of main configuration scripts...

Reply:
Then there is probably no point having winconf-<compiler>.cmake files.
Did you noticed src/app/CMakeLists.txt doesn't contain specific compiler flags? well I find that neat... anyway I care less about cmake so I won't insist.

Reply:
>Then there is probably no point having winconf-<compiler>.cmake files.

IMO, qBittorrent CMake configuration is too fragmented. I'm playing with CMake a little bit, so maybe I'll fix something else in the future.